jenkins/manifests-update.jenkinsFile

```diff
@@ -0,0 +1,44 @@
+pipeline {
+    options {
+        timeout(time: 1, unit: 'HOURS')
+        buildDiscarder(logRotator(daysToKeepStr: '7'))
+    }
+    agent none
+    triggers {
+        cron('H 0 * * *')
+        }
+    stages {
+        stage('Update Manifests') {
+            agent {
+                docker {
+                    label 'Jenkins-Agent-AL2-X64-C54xlarge-Docker-Host'
+                    image 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2'
+                    args '-e JAVA_HOME=/opt/java/openjdk-11'
+                    registryUrl 'https://public.ecr.aws/'
+                    alwaysPull true
+                }
+            }
+            steps {
+                script {
+                    withCredentials([usernamePassword(credentialsId: 'jenkins-github-bot-token', passwordVariable: 'GITHUB_TOKEN', usernameVariable: 'GITHUB_USER')]) {
+                        sh """
+                        git remote set-url origin https://opensearch-ci:${GITHUB_TOKEN}@github.com/opensearch-project/opensearch-build
+                        git config user.email "opensearch-infra@amazon.com"
+                        git config user.name "opensearch-ci"
+                        git checkout -b update-manifest-${BUILD_NUMBER}
+                        ./manifests.sh update
+                        git add . && git commit -sm "Update manifests"
```

Comment:
What happens when there are not changes to add or commit?

Reply:
It throws an exception. Let me put a try catch block to handle that and also lessen the scope of github token

Reply:
Added a try catch block that will handle the exception if anything goes wrong and exit gracefully if there is nothing to commit.
Thanks!